docs/conf.py

```diff
@@ -115,6 +135,14 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'FireSimdoc'
 
+html_context = {
+    "version": version
+}
+
+# can put custom strings here that are generated from this file
```

Comment:
It's not clear to me what this does. Is it appended to the end of all .rst files as a means of injecting common code?

Reply:
Correct: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-rst_epilog

Reply:
I'll update the comment.